### ISSUE TITLE
Provide more contrast to the recording overlay

### ISSUE THREAD

**Issue by stephenmeszaros** [wulkano/kap#201]
The dotted frame lines that are displayed—representing the recording area—can be very hard to parse at times. Sometimes I have to hunt around the screen for them, which feels like work. Perhaps a dark overlay or scrim within the dotted frame lines could help?

Happy to help with the design here if others agree. Posting in response to some ❤️ in a conversation on Slack.

**Comment by skllcrn** [comment]
Thanks a bunch mate. We completely agree, the intention is for there to be a dark overlay surrounding the selected area. 

![app selection recording ready](https://cloud.githubusercontent.com/assets/8440026/26639293/b5bc72fe-4624-11e7-8f62-2a2bdd3264ed.png)

There's some challenges with the implementation of it that might not be possible to work around within the constraints of our current solution.

**Comment by skllcrn** [comment]
https://github.com/wulkano/kap/pull/409

**Comment by skllcrn** [comment]
Closing in favour of #113
